Sources/XiEditor/Theme.swift

```diff
@@ -82,7 +93,8 @@ extension Theme {
         let caret = NSColor(jsonRgbaColor: json["caret"] as? [String: Any] ?? [:])
         let line_highlight = NSColor(jsonRgbaColor: json["line_highlight"] as? [String: Any] ?? [:])
 
-        let find_highlight: NSColor? = NSColor(jsonRgbaColor: json["find_highlight"] as? [String: Any] ?? [:])
+        // Use default find highlight color instead of theme's find_highlight value
+        let find_highlight: NSColor? = Theme.defaultFindHighlightColor
```

Comment:
I think if the theme specifies a color for find highlights, then it's ok to use that color instead of forcing the default yellow.

Reply:
Sure, I'm also bit not clear on expected behavior. I will wait for clarification https://github.com/xi-editor/xi-mac/issues/470#issuecomment-544213362